setup.cfg

```diff
@@ -33,11 +33,13 @@ install_requires=
     pre-commit
     pytest
     pytest-rerunfailures
-    selenium
+    selenium==3.141.0
```

Comment:
any special reason we are pinning selenium latest version?
I think it better to ping like `selenium>=3.141.0` so we can accept upcoming versions.

@sshveta You can make this as part of the next PR.